Gemfile.lock

```diff
@@ -1,15 +1,37 @@
 PATH
   remote: .
   specs:
-    sql_search_parser (0.0.4)
+    sql_search_parser (1.0.0)
 
 GEM
   remote: http://rubygems.org/
   specs:
+    ast (2.4.2)
+    json (2.6.3)
     minitest (5.3.1)
+    parallel (1.22.1)
+    parser (3.2.0.0)
+      ast (~> 2.4.1)
     racc (1.4.11)
+    rainbow (3.1.1)
     rake (10.1.1)
+    regexp_parser (2.6.1)
     rexical (1.0.7)
+    rexml (3.2.5)
+    rubocop (1.41.1)
+      json (~> 2.3)
+      parallel (~> 1.10)
+      parser (>= 3.1.2.1)
+      rainbow (>= 2.2.2, < 4.0)
+      regexp_parser (>= 1.8, < 3.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.23.0, < 2.0)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (>= 1.4.0, < 3.0)
+    rubocop-ast (1.24.1)
+      parser (>= 3.1.1.0)
+    ruby-progressbar (1.11.0)
+    unicode-display_width (2.4.2)
```

Comment:
Hey Pavel, thanks for resolving the previous concern. I have one more question: Has this new update been tested by bundling the gems on the rtb-trader app? does it works fine or have any conflicts there?

Reply:
`Gemfile.lock` file of the gem is ignored when installing the gem in a rails application so it won't affect rtb-trader. `Gemfile.lock` is needed only for the convenience of the gem contributors.